website/app/components/doc/wcag-list/index.js

```diff
@@ -1,810 +1,817 @@
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-4-1-2',
+        title: 'Name, Role, Value (Level A)',
+        number: '4.1.2',
+        url: 'https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html',
+        level: 'A',
+        description:
+          'For all user interface components, the name and role can be programmatically determined; states, properties, and values that can be set by the user can be programmatically set; and notification of changes to these items is available to user agents, including assistive technologies.',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-4-1-3',
+        title: 'Status Messages (Level AA)',
+        number: '4.1.3',
+        url: 'https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html',
+        level: 'AA',
+        description:
+          'In content implemented using markup languages, status messages can be programmatically determined through role or properties such that they can be presented to the user by assistive technologies without receiving focus.',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-2-1',
+        title: 'Conformance Level',
+        number: '5.2.1',
+        url: 'https://www.w3.org/TR/WCAG21/#cc1',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-2-2',
+        title: 'Full pages',
+        number: '5.2.2',
+        url: 'https://www.w3.org/TR/WCAG21/#cc2',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-2-3',
+        title: 'Complete processes',
+        number: '5.2.3',
+        url: 'https://www.w3.org/TR/WCAG21/#cc3',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-2-4',
+        title: 'Only Accessibility-Supported Ways of Using Technologies',
+        number: '5.2.4',
+        url: 'https://www.w3.org/TR/WCAG21/#cc4',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-2-5',
+        title: 'Non-Interference',
+        number: '5.2.5',
+        url: 'https://www.w3.org/TR/WCAG21/#cc5',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-3-1',
+        title: 'Required Components of a Conformance Claim',
+        number: '5.3.1',
+        url: 'https://www.w3.org/TR/WCAG21/#conformance-required',
+        description: '',
+      },
+      {
+        type: 'success-criteria',
+        id: 'wcag-5-3-2',
+        title: 'Optional Components of a Conformance Claim',
+        number: '5.3.2',
+        url: 'https://www.w3.org/TR/WCAG21/#conformance-optional',
+        description: '',
+      },
+    ];
+    let criteriaList = this.args.criteriaList;
+    let filteredCriteria = criteria.filter((criterion) =>
+      criteriaList.includes(criterion.number)
+    );
+    return filteredCriteria;
```

Comment:
possible improvements:
- you don't need to have a `criteriaList` variable, if you use it only once in the code, you can use directly the `this.args.criteriaList` array (which is already in memory)
- (if you want) you don't need to create a temporary `filteredCriteria`, you can directly return the result of the `filter` method on the original array (it already creates a new array) 
```suggestion
    return criteria.filter((criterion) =>
      this.args.criteriaList.includes(criterion.number)
    );
```

Reply:
@didoo I was actually kind of confused about the conditions under which it's okay to filter the original array, I'll DM you.